core-services/prow/02_config/_config.yaml

```diff
@@ -58,13 +58,6 @@ branch-protection:
       repos:
         toolchain-operator:
           protect: false
-    integr8ly:
-      repos:
-        integreatly-operator:
-          exclude:
-          - master
-          - release-*
-          protect: false
```

Comment:
This does the opposite of what you describe in the PR - this will make us try to set up BP on that repo

Reply:
Thanks! Updated.